e2e/app/page/dataset-build-page.ts

```diff
@@ -16,9 +16,9 @@ import WorkspaceAnalysisPage from './workspace-analysis-page';
 import WorkspaceAboutPage from './workspace-about-page';
 import ExportToNotebookModal from 'app/modal/export-to-notebook-modal';
 import { getPropValue } from 'utils/element-utils';
-import { LinkText } from 'app/text-labels';
+import { CohortsSelectValue, ConceptSetSelectValue, LinkText } from 'app/text-labels';
 
-const pageTitle = 'Dataset Page';
+const pageTitle = '(Edit )?Dataset( Page)?';
```

Comment:
Regex to make the `dataset-build-page.ts` class more generic. `dataset-edit-page.ts` will not be needed.